tests/test_cmdline.py

```diff
@@ -31,6 +31,19 @@ def test_sig(sig_name):
 @helpers.filtered_test
 @pytest.mark.parametrize('sig_stfl_name', helpers.available_sig_stfls_by_name())
 def test_sig_stfl(sig_stfl_name):
+    # Define the list of LMS varients to allow
+    names_to_allow = ["LMS_SHA256_H5_W1", "LMS_SHA256_H5_W2", 
```

Comment:
Should this list be documented somewhere, e.g., in the README and/or docs/algorithms/sig_stfl as otherwise users can easily complain about certain (many) SHBS algs not being available for testing?